tests/integration/helm_default_installation_test.go

```diff
@@ -52,19 +53,15 @@ func Test_Helm_Default(t *testing.T) {
 				require.Len(t, secret.Data, 10)
 				return ctx
 			}).
-		Assess("3 fluentd logs pods are created and running",
+		Assess("fluentd logs pods are available",
 			func(ctx context.Context, t *testing.T, envConf *envconf.Config) context.Context {
-				require.Eventually(t, func() bool {
-					pods := k8s.ListPods(t, ctxopts.KubectlOptions(ctx), v1.ListOptions{
-						LabelSelector: fmt.Sprintf("app=%s-sumologic-fluentd-logs", releaseName),
-						FieldSelector: "status.phase=Running",
-					})
-					return len(pods) == 3
-				}, waitDuration, tickDuration)
-
+				filters := v1.ListOptions{
+					LabelSelector: fmt.Sprintf("app=%s-sumologic-fluentd-logs", releaseName),
+				}
+				utils.WaitUntilPodsAvailable(t, ctxopts.KubectlOptions(ctx), filters, 3, waitDuration, tickDuration)
```

Comment:
Plz not `utils` 🙏

Reply:
Come up with a better name then? `k8s` like in terratest?

Reply:
Changed to `waituntil`